src/app/admin/faculty/faculty.component.scss

```diff
@@ -0,0 +1,63 @@
+body {
+  font-size: 1rem;
+  line-height: 1.6;
+}
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+.CreateForm {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 30px 0px;
+  border-radius: 25px;
+  padding: 20px;
+}
+
+.btn {
+  width: 200px;
+}
+
+a:hover {
+  cursor: pointer;
+}
+
+.material-icons.delete:hover {
+  color: red;
+}
+
+.material-icons.edit:hover {
+  color: orange;
+}
+.Form {
+  width: 50%;
+  margin: 20px 0px;
+}
+
+table {
+  width: 100%;
+}
+
+.Form {
```

Comment:
.form. Should be combined with the previous .form selector. Class name should be specific, like bla-bla-form